src/ring_theory/ideal/operations.lean

```diff
@@ -25,20 +25,18 @@ variables [comm_semiring R] [add_comm_monoid M] [module R M]
 
 open_locale pointwise
 
-instance has_smul' : has_smul (ideal R) (submodule R M) :=
-⟨submodule.map₂ (linear_map.lsmul R M)⟩
+instance has_smul' : has_smul (ideal R) (submodule R M) := ⟨submodule.map₂ (linear_map.lsmul R M)⟩
```

Comment:
For definitions and instances like this I'd prefer to have the body on its own line. (I'd say ideally we'd have at most one "new idea" per line.)
```suggestion
instance has_smul' : has_smul (ideal R) (submodule R M) :=
⟨submodule.map₂ (linear_map.lsmul R M)⟩
```